### ISSUE TITLE
Do not show clothes in inventory

### ISSUE THREAD

**Issue by mazes-80** [sfence/clothing#5]
All those items shouldn't appear in inventory as they can only be crafted via machines
sed -i '/groups =/s/}/, not_in_creative_inventory=1}/' clothes.lua

As they are not nodes no matter about creative mode.

May add a variable to settingstype.txt to force visibility.

One other approach would be to remove descriptions, but it wouldn't allow adding a setting type

**Comment by sfence** [comment]
Thanks for the report.

I will look at it.

**Comment by sfence** [comment]
Should be fixed with release 2.3.2.

**Comment by Skivling** [comment]
@sfence 
I don't think this is fixed. Or maybe my problem is slightly different. But there should be a way to make the clothing not appear in the inventory item lists at all, as they take up more than 20 pages. The current settings page does not have a value that fixes this.

**Comment by sfence** [comment]
@Skivling It is the opposite problem, you are talking about.

I do not know about settings/group which allow that. But is a different issue.

**Comment by mazes-80** [comment]
sfence: they should just belong to group not_in_creative_inventory

> sed -i '/groups =/s/}/, not_in_creative_inventory=1}/' clothes.lua

they may however stay in inventory in creative mode.
maybe just add a variable checking creative, then use it for the value of not_in_creative_inventory

**Comment by sfence** [comment]
@mazes-80 

Group `not_in_creative_inventory` causes recipes not to be shown in inventory mods as `inventory_plus`.

Variable checking will not work with players with creative privilege.

**Comment by mazes-80** [comment]
> Group `not_in_creative_inventory` causes recipes not to be shown in inventory mods as `inventory_plus`.

That's exactly what I point: why things that are not meant to be crafted via inventory still appear in inventory ?

> Variable checking will not work with players with creative privilege.

It seems more related to an issue in inventory mod. No idea how to deal with it from any other mod.

**Comment by sfence** [comment]
`inventory_plus` normally shows recipes for `technic` mod machines. It is a benefit of using some mod like `inventory_plus`, that you can also see non-inventory recipes.

So, you are looking for some settings that allow you to disable registration of clothes machine recipes in `inventory_pkus` and potentially similar mods?
